src/org/benf/cfr/reader/bytecode/analysis/structured/statement/StructuredTry.java

```diff
@@ -77,8 +78,13 @@ public Dumper dump(Dumper dumper) {
             dumper.separator(")");
         }
         tryBlock.dump(dumper);
-        for (Op04StructuredStatement catchBlock : catchBlocks) {
+        dumper.removePendingCarriageReturn().separator(" ");
```

Comment:
Lol - What I meant was - there's no need to mess around with iterators - your code is equivalent to

        tryBlock.dump(dumper);
        for (Op04StructuredStatement catchBlock : catchBlocks) {
            dumper.removePendingCarriageReturn().separator(" ");
            catchBlock.dump(dumper);
        }
        if (finallyBlock != null) {
            dumper.removePendingCarriageReturn().separator(" ");
            finallyBlock.dump(dumper);
        }